compair/static/modules/user/user-module.js

```diff
@@ -347,6 +370,50 @@ module.controller("UserCourseController",
             }
         };
 
+        $scope.updateLTIList = function() {
```

Comment:
better to pre-load this (make it conditional in preloader with `AuthTypesEnabled.lti`)